src/Plugin/CaptureUtility/ScreenshotCaptureUtility.php

```diff
@@ -39,4 +55,29 @@ public function getResponse() {
     return $this->response;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function missingDependencies() {
+    $capture_message = $this->t('Screen Capture package missing.');
```

Comment:
I see it in the base now.

It may not be necessary to perform translation in this method but instead in `WebPageArchiveQueueForm` where `$this->t` is already available. It would be more efficient than _always_ translating the strings, but by not _always_ translating those strings but interpreting the array we wouldn't have complete string discovery. The messages shouldn't be seen at all, though, unless someone had a bad install.

Thoughts?